hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneKeyDetails.java

```diff
@@ -21,7 +21,10 @@
 import org.apache.hadoop.fs.FileEncryptionInfo;
```

Comment:
Import for `@JsonIgnore`:

```suggestion
import com.fasterxml.jackson.annotation.JsonIgnore;
import org.apache.hadoop.fs.FileEncryptionInfo;
```